test/support/rake_helpers.rb

```diff
@@ -25,13 +25,13 @@ def test_files
   end
 
   def all_test_files
-    activerecord_test_files = Dir.
-      glob("#{ARTest::CockroachDB.root_activerecord}/test/cases/**/*_test.rb").
-      reject { _1.match?(%r(/adapters/(?:mysql2|sqlite3)/)) }.
-      prepend(COCKROACHDB_TEST_HELPER)
+    activerecord_test_files =
```

Comment:
The file order did matter, I opened a related issue in rails (https://github.com/rails/rails/issues/50768)